src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+$width: 200px;
+$color: #00acdc;
+$color1: #060b35;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+h1 {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+.card {
+  width: #{$width};
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    color: #{$color1};
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    margin: 0 0 16px;
+  }
+
+  &__button {
+    display: flex;
+    text-decoration: none;
+    text-transform: uppercase;
+    box-sizing: border-box;
+    color: #fff;
+    align-items: center;
+    justify-content: center;
+    font-size: 14px;
+    font-weight: 700;
+    height: 40px;
+    border: 1px solid #{$color};
+    border-radius: 5px;
+    background: #{$color};
+  }
+
+  &__button:hover {
+    background-color: #fff;
+    color: #{$color};
+  }
+
+  &__product-code {
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    margin: 0 0 17px;
+  }
+}
+
+.price {
+  margin-bottom: 16px;
+  color: #616070;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.text {
+  color: #616070;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.rewiew {
+  color: #{$color1};
+  font-size: 10px;
+  font-weight: 400;
```

Comment:
remove it

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3622/files#r1328057841
still not fixed